bin/main.js

```diff
@@ -24,8 +26,18 @@ export async function main(argv) {
   const sessionId = uuid();
   const env = getEnv();
   const log = createLogger(sessionId, env);
+
   const packagePath = await pkgUp(); // the user's own package.json
+  if (!packagePath) {
+    log.error(noPackageJson());
+    process.exit(253);
```

Comment:
What does this number represent (253)?

Reply:
Nothing, I just picked one that wasn't used yet.

Reply:
I'm thinking in a future PR maybe we should assign constants to these exit codes, so we can have a list of what they mean and know which ones are used or not.